### PR TITLE
Add Personal Access Token document

### PR DESCRIPTION
**Pull request by MarkvanMents**
Added in Developer Portal Guide > Colaboration as it is mainly to do with using Warden

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/app-repository-api.md [hunk 1]
```diff
@@ -31,14 +31,7 @@ All available endpoints are shown in the following table:
...
 
-You can manage your Mendix personal access tokens via [Warden](https://warden.mendix.com/) as follows:
-
-1. Go to [Warden](https://warden.mendix.com/).
-2. Click **Add**.
-3. Enter the **Name** of the token.
-4. Choose at least this scope: `mx:modelrepository:repo:read`.
-5. Click **Create**. A pop-up window opens and shows the Personal Access Token.
-6. Keep the token safe. It is only displayed once. After you close the pop-up window, you cannot see it anymore.
+You can manage your Mendix personal access tokens via [Warden](/developerportal/collaborate/warden)
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/apidocs/app-repository-api.md:34]: Need a period.

File: content/data-hub/data-hub-catalog/register-data.md [hunk 2]
```diff
@@ -65,31 +65,7 @@ The [Data Hub Registration API specification](https://datahub-spec.s3.eu-central
...
-    The token is generated and displayed in a pop-up window:
-
-    ![generated token](attachments/register-data/generated-token.png)
-
-6. Copy the **Token secret** and keep this token in a secure place. You will not get another chance to view this token once you close this pop-up window.
-
-7. Click **Close** to return to the **Warden** homepage where all your Personal Access Tokens are listed. If needed, you can delete your token from this list using the red trash bin button.
-
-    ![generated token](attachments/register-data/warden-home-with-token.png)
+You can create a Personal Access Token in the Mendix [Warden](/developerportal/collaborate/warden) application. Follow the steps below:
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/register-data.md:68]: `Follow the steps below:` to be deleted?

File: content/developerportal/collaborate/warden.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+tags: ["Personal Access Token", "API", "PAT", "Warden", "Developer Portal"]
+---
+
+## 1 Introduction 
+
+This document explains the concept of Personal Access Tokens (PATs), how platform users can obtain one using **Warden**, and how platform users can use these security tokens to give application access to Mendix platform services on their behalf.
+
+The following Mendix services support usage of PATs:
+
+* [DataHub Catalogue API](/apidocs-mxsdk/apidocs/data-hub-apis)
```
Comment (Adam-Dupaski) [inline content/developerportal/collaborate/warden.md:15]: Please fix spelling on `DataHub Catalogue`

Comment (Adam-Dupaski) [reply content/developerportal/collaborate/warden.md:15]: Please take a look for other inconsistent spellings throughout.

File: content/data-hub/data-hub-catalog/register-data.md (hunk 2 above)
Comment (johannahemminger) [reply content/data-hub/data-hub-catalog/register-data.md:68]: Looks good just to link to the new doc. +1 to Adam's comment!

File: content/developerportal/collaborate/warden.md (hunk 3 above)
Comment (MarkvanMents) [reply content/developerportal/collaborate/warden.md:15]: Sorry - they didn't look wrong to me when I proofread it 😁

**Comment by MarkvanMents** [comment]
PM (Jaap) approved over Slack.

**Pull request merged**